web/src/wallet/config/wallet.config.ts

```diff
@@ -12,7 +12,7 @@ export type Checker = {
   buyToSymbol: string
 }
 
-const rpcNetworkList = {
+export const rpcNetworkList = {
   florencenet: {
```

Comment:
nit 
**export** const rpcNetworkList  

export can be removed, currently rpcNetworkList is not used outside this file